test/dump.cpp

```diff
@@ -869,11 +870,23 @@ dump_parser_sequence_info (const VkParserSequenceInfo * info)
   end_object ();
 }
 
+static void
+dump_parser_vk_pic_entry (const VkPicIf * entry)
+{
+  if (entry) {
+    start_object ("VkPicIf");
+    print_integer ("decodeWidth", entry->decodeWidth);  // frame_num(short-term) or LongTermFrameIdx(long-term)
+    print_integer ("decodeHeight", entry->decodeHeight);  // 0=short term reference, 1=long term reference
+    print_integer ("decodeSuperResWidth", entry->decodeSuperResWidth);  // non-existing reference frame (corresponding PicIdx
+    end_object ();
+  }
+}
+
```

Comment:
Just for the sake of "atomicity and clearness"  I would move these changes in another commit.